.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules/
 dist/
 npm-debug.log
 *.env
+.idea/
```

Comment:
@cadecairos what's this `idea` directory about?

Reply:
IntelliJ editor file (an annoyingly auto-generated file that we don't want on the repo)

Reply:
ah kk